.github/workflows/gh_runner-scheduler.yml

```diff
@@ -0,0 +1,48 @@
+name: GitHub Runner Job AutoScaler
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 09 * * MON-FRI'
+    - cron: '0 18 * * MON-FRI'
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  resource-group: 'io-p-github-runner-rg'
+  job-name: 'io-infra-github-runner-job'
+
+jobs:
+  scheduler:
+    runs-on: ubuntu-latest
+    environment: prod-runner
+    env:
+      MAXIMUM_REPLICA: 10
```

Comment:
can we set this parameter as env variable? so we have all variables in the same point